### PR TITLE
add *.out in .gitignore

### PR DESCRIPTION
**Pull request by jwpark-sungshin**
exclude *.out from git

resolved: TEST-1234

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*.out
```
Comment (pjbear1215) [inline .gitignore:1]:
```suggestion
*.out
*.o
*.exe
```

**Review by pjbear1215 — CHANGES_REQUESTED** [review]
This is looking great. I've left a few comments that should be addressed before this gets merged.

File: .gitignore (hunk 1 above)
Comment (jwpark-sungshin) [reply .gitignore:1]: OK.

**Review by pjbear1215 — APPROVED** [review]
Well done!

**Pull request closed without merging**